convert-hf-to-gguf.py

```diff
@@ -427,9 +427,6 @@ def get_vocab_base_pre(self, tokenizer) -> str:
         # NOTE: if you get an error here, you need to update the convert-hf-to-gguf-update.py script
         #       or pull the latest version of the model from Huggingface
         #       don't edit the hashes manually!
-        if chkhsh == "0ef9807a4087ebef797fc749390439009c3b9eda9ad1a097abbe738f486c01e5":
```

Comment:
Not really sure why this happened...
I ran convert-hf-to-gguf-update.py as instructed

Reply:
Most likely you don't have access to the HF repos